timeseries/src/autogluon/timeseries/models/autogluon_tabular/transforms.py

```diff
@@ -13,29 +13,32 @@
     TIMESTAMP,
     TimeSeriesDataFrame,
 )
-from autogluon.timeseries.transforms.target_scaler import LocalTargetScaler, get_target_scaler
+from autogluon.timeseries.transforms.target_scaler import TargetScaler, get_target_scaler
 
 from .utils import MLF_ITEMID, MLF_TIMESTAMP
 
 
 class MLForecastScaler(BaseTargetTransform):
-    def __init__(self, scaler_type: Literal["standard", "robust", "min_max", "mean_abs"]):
+    def __init__(self, scaler_type: Literal["standard", "min_max", "mean_abs", "robust"]):
         # For backward compatibility
-        self.scaler_type = scaler_type
-        self.ag_scaler: Optional[LocalTargetScaler] = None
+        self.scaler_type: Literal["standard", "min_max", "mean_abs", "robust"] = scaler_type
```

Comment:
Why is this annotation necessary?

Reply:
it's not necessary per se but also not bad?